ui/webui/resources/fonts/manrope.css

```diff
@@ -0,0 +1,55 @@
+/* manrope-100 */
```

Comment:
Does this file actually get used? Or is it included in case a future UI wants to explicitly use Manrope?

Reply:
Yes, for explicit use.